spec/features/cart/user_can_delete_item_from_cart_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.feature "user visits cart" do
+  before(:each) do
+    @course1 = create(:course, id: 1)
+    @course2 = create(:course, id: 2)
+    @cart = Cart.new({"1" => 2, "2" => 3})
+  end
+
+  scenario "can delete course from cart and link back to item" do
```

Comment:
@samanthagongora 
I had to add the following code to add the cart to session: 

    visit root_path
    click_button "Add to Cart", match: :first